Source/EasyNetQ/RabbitHutch.cs

```diff
@@ -387,7 +387,7 @@ public static IBus CreateBus(ConnectionConfiguration connectionConfiguration, Ad
             registerServices(container);
             ComponentRegistration.RegisterServices(container);
 
-            return container.Resolve<IBus>();
+            return container.Resolve<RabbitBus>();
```

Comment:
I saw now that the factory still return an `IBus`, but anyway, I would to think more on the drawbacks of this.